recipes/configs/llama2/13B_lora.yaml

```diff
@@ -75,6 +72,7 @@ loss:
 # Training
 epochs: 1
 max_steps_per_epoch: null
+gradient_accumulation_steps: 1
```

Comment:
Gradient accumulation is not enabled on LoRA distributed recipe yet

Reply:
#644

Reply:
![Screenshot 2024-04-03 at 10 57 53 AM](https://github.com/pytorch/torchtune/assets/20175092/343a2465-b2a1-4cff-a04b-9a1b23b8ee84)

cc @RdoubleA

Reply:
This is why we should just enable it for the recipe, right? (Agree we still need to handle this missing/erroneous field issue more gracefully)